### DISCUSSION TITLE
Customize Scroll bar for browser

### DISCUSSION THREAD

**Discussion by LeviticusNelson** [mantinedev/mantine#2258] [Feature requests]
I know that there is a way to style the ```<ScrollArea />``` track and thumb for the scrollbar, but there is not an easy way to style ``` ::-webkit-scrollbar``` and others without injecting global css values.

**Comment by rtivital** [comment]
You can use Global component for that. If we provide anything like that, then it will add the same global styles it would be just hidden from you.

**Reply by LeviticusNelson** [reply]
Thanks for the reply! I just learned Global component a little better, and now I got rid of my global.css for other stuff as well!
